compiler/luci/pass/src/QuantizationUtils.cpp

```diff
@@ -83,6 +83,7 @@ void compute_asym_scale_zp(float min, float max, float &scaling_factor, int64_t
     assert(min >= 0 && max >= 0);
     nudged_zero_point = kMinScale;
     scale = max / (qmax_double - qmin_double);
+    rmin = 0;
```

Comment:
If min >= 0 (according to the assert in line 83), `rmin` would be already 0 due to line 65. Do we have to set rmin again?

If rmin is not changed, we can keep `const` in line 65~66.

Reply:
oh, I didn't catch it. Thank you.